### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1228] [Announcements]
## What's Changed
* Fix buffer deletion with bufferline
* Align tab switching in neo-tree with rest of the UI (`H` and `L`)
* Add `O` to open file in `neo-tree` with external applications (Mac and Linux only)
* Add `<leader>C` binding to force close buffer (useful for terminals)
* Various code clean up

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v2.5.0...v2.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by vtvz** [comment]
And how now I can show hidden files in neo-tree?

**Reply by mehalter** [reply]
> And how now I can show hidden files in neo-tree?

This was a complete oversight on my part! So sorry! Someone thankfully contributed a PR to resolve this and map toggling hidden files to `h`. This addition is currently in `nightly` but I will get a new patch release tagged today to fix this. Sorry again!!

**Reply by mehalter** [reply]
@vtvz
